### DISCUSSION TITLE
Login fails at Realm

### DISCUSSION THREAD

**Discussion by Conan1996-2** [azerothcore/azerothcore-wotlk#14477] [General]
First time trying out my own personal private server to play around in. I followed the directions given via https://www.azerothcore.org/wiki/installation#azerothcore-classic-setup
for Ubuntu(Linux).

I have created an account for the client and logged in (Tested with and without the correct password and it works as intended for both)

Problem is, it lists the default azerothserver, and I can not log into it. It will say that its logging into the server, then goes back to the realm list.

I have both ./authserver and ./worldserver running in two separate shell windows as well, with no errors.

Any help on what I did wrong or may be missing please?

**Comment by nephente** [comment]
I got the same problem, but i compiled on Debian 11.
The name of the server is yellow when show in the list, in a precompiled servers that i test normally is green
Any idea ?

**Reply by nephente** [reply]
Ok, is a good idea finish to read all the guide, my server is running on a VM in to a NAS Server, the is necessary change the ip on data base


    MySQL CLI Commands (This step is not needed if you used a MySQL Manager like HeidiSQL)
        $ sudo mysql
        You should see a prompt change to mysql>
        use acore_auth;
        Replace your IP with the one you've chosen to use from above
        UPDATE realmlist SET address = '[your_ip]' WHERE id = 1;
        exit

**Reply by Conan1996-2** [reply]
I actually figured out my issue: It was a spectrum provider issue. Their router doesn't allow you to use WAN address to access a local device if you're trying to access it from that same network. Odd, but if I used a VPN to emulate another network, It would work. If this is not clear enough, If the DATABASE realmlist and  was set up as 127.0.0.1 and the CLIENT realmlist was set up to the server, I would have no issues accessing the server. If they were both set as the MODEM address, I couldn't access unless I VPN out of my local network.
